src/sidebar/services/load-annotations.ts

```diff
@@ -209,7 +215,7 @@ export class LoadAnnotationsService {
     // configure the connection to the real-time update service to send us
     // updates to any of the annotations in the thread.
     if (!isReply(annotation)) {
-      const id = /** @type {string} */ (annotation.id);
+      const id = annotation.id!;
```

Comment:
Presumably since we've retrieved the `annotation` from the service, it will have an ID. I'm just perpetuating the existing assumption here...